web-src/src/pages/PageComposer.vue

```diff
@@ -0,0 +1,101 @@
+<template>
+  <div>
+    <tabs-music></tabs-music>
```

Comment:
We don't show the tabs on e. g. the artist page, please drop this line (might also have to update the flags in the router)